### PR TITLE
FRONTEND: updated hearbeat feature

### PR DESCRIPTION
**Pull request by jelyoussefi**

### CODE REVIEW

**Comment by Cloud-Chen** [comment]
Can one of the admins verify this patch?

File: public-interface/scripts/wait-for-heartbeat.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) 2014 Intel Corporation
```
Comment (wagmarcel) [inline public-interface/scripts/wait-for-heartbeat.js]: Pleas update the copyright year to 2018

**Comment by nevets963** [comment]
@jelyoussefi Looks good, I'm just thinking through if other repos will also need this? If that is the case, then those images will also have to contain nodejs (not a big deal, but worth noting)

**Comment by nevets963** [comment]
@jelyoussefi I assume this can now be approved, now that PR https://github.com/Open-IoT-Service-Platform/platform-launcher/pull/35 has been approved?

**Pull request merged**